src/hypertable_cache.h

```diff
@@ -12,7 +12,10 @@
 #include "cache.h"
 #include "hypertable.h"
 
-extern TSDLLEXPORT Hypertable *ts_hypertable_cache_get_entry(Cache *cache, Oid relid);
+extern TSDLLEXPORT Hypertable *ts_hypertable_cache_get_entry(Cache *cache, Oid relid,
+															 bool missing_ok);
+extern TSDLLEXPORT Hypertable *ts_hypertable_cache_get_entry_missing_ok(Cache *cache, Oid relid);
```

Comment:
hmm i dont like those function names and I would prefer to have the missing_ok be a bool argument to the function call

Reply:
> hmm i dont like those function names and I would prefer to have the missing_ok be a bool argument to the function call

It's one of the most common case. I wanted to use the argument first, but then I realised that it is too many cases (renaming is automatic :). So I feel that having the wrapper is better. I am not sure what the function name will be better. Let me know if you have ideas for the name.

Reply:
I went for argument `missing_ok` to be always required.